packages/core/src/shared/extensionUtilities.ts

```diff
@@ -254,6 +262,28 @@ export function showWelcomeMessage(context: vscode.ExtensionContext): void {
     }
 }
 
+function _getDocumentationUrl(urls: { cloud9: vscode.Uri; toolkit: vscode.Uri }): vscode.Uri {
+    return isCloud9() ? urls.cloud9 : urls.toolkit
```

Comment:
rather than having to do vscode.Uri.parse everywhere you could just have c9/toolkit be strings and do the parsing in here I think

Reply:
I was thinking that eventually I think we'll get rid of C9 and just directly use the variable in the constants file, so all these functions would be deleted